pkgs/applications/video/davinci-resolve/default.nix

```diff
@@ -0,0 +1,169 @@
+{ stdenv
+, lib
+, cacert
+, curl
+, runCommandLocal
+, targetPlatform
+, unzip
+, appimage-run
+, addOpenGLRunpath
+, libGLU
+, alsaLib
+, xlibs
+, xorg
+, buildFHSUserEnv
+, bash
+, writeText
+, ocl-icd
+, xkeyboard_config
+, glib
+, libarchive
+}:
+let
+  davinci = (
+    stdenv.mkDerivation rec {
+      pname = "davinci-resolve";
+      version = "17.4.1";
```

Comment:
Updated to 17.4.2, see https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=davinci-resolve

Reply:
Thanks. Updated!